libretro/libretro_core_options.h

```diff
@@ -114,28 +114,36 @@ struct retro_core_option_definition option_defs_us[] = {
     {
         CORE_NAME "-169screensize",
         "16:9 Resolution",
```

Comment:
Maybe turn it "Wide Resolutions"

Reply:
Yeah, I like that.

I think it should be "Wide Resolution" (without "s") to be consistent with the "4:3 Resolution" option. The other option would be to change both ("4:3 Resolution" and "Wide Resolution")

Reply:
Sure